src/main/java/org/gridsuite/modification/server/NetworkModificationException.java

```diff
@@ -79,19 +81,19 @@ Type getType() {
         return type;
     }
 
-    public static NetworkModificationException createEquipmentTypeUnknown(String type) {
+    public static NetworkModificationException createEquipmentTypeUnknown(@NonNull String type) {
         Objects.requireNonNull(type);
```

Comment:
with the @NonNull annotation, you can remove Objects.requireNonNull call

Reply:
Done